packages/modules/web_themes/koala/source/src/components/BaseTable.vue

```diff
@@ -28,99 +30,196 @@
               class="search-field white-outline-input"
               input-class="text-white"
             >
-              <template v-slot:append>
+              <template #append>
                 <q-icon name="search" color="white" />
               </template>
             </q-input>
           </div>
         </div>
       </template>
 
-      <!-- Dynamic slot for custom cell rendering -->
+      <!-- header ----------------------------------------------------------->
+      <template v-if="props.rowExpandable" #header="header">
+        <q-tr :props="header">
+          <!-- space for arrow column -->
+          <!-- the abbreviation col has to remain as it comes from Quasar QTableSlots.header -->
+          <q-th auto-width :props="{ ...header, col: {} }" />
+          <!-- the other columns -->
+          <q-th
+            v-for="column in header.cols"
+            :key="column.name"
+            :props="{ ...header, col: column }"
+          >
+            {{ column.label }}
+          </q-th>
+        </q-tr>
+      </template>
+
+      <!-- body ------------------------------------------------------------->
+      <!-- "cols" and "col" must match Quasar QTable slot prop names and can't be renamed to "columns"/"column" -->
+      <template v-if="props.rowExpandable" #body="rowProps: BodySlotProps<T>">
+        <q-tr
+          :key="`main-${rowProps.key}`"
+          :props="rowProps"
+          @click="onRowClick($event, rowProps.row)"
+          class="clickable"
+        >
+          <q-td auto-width>
+            <q-btn
+              dense
+              flat
+              round
+              size="sm"
+              :icon="
+                rowProps.expand ? 'keyboard_arrow_up' : 'keyboard_arrow_down'
+              "
+              @click.stop="rowProps.expand = !rowProps.expand"
+            />
+          </q-td>
+
+          <template v-for="col in rowProps.cols" :key="col.name">
```

Comment:
Hier wird eine eigene Variable definiert, die durchaus ausgeschrieben werden kann.
```suggestion
          <template v-for="column in rowProps.cols" :key="column.name">
```
Die Änderung erforder noch weitere Anpassungen in den folgenden Zeilen bis Z106.